### ISSUE TITLE
How do I use custom `accessToken` and `expires` properties with credentials provider?

### ISSUE THREAD

**Issue by frencojobs** [iaincollins/next-auth#359]
**Your question**
How do I use custom `accessToken` and `expires` properties when using the `credentials` provider?

**What are you trying to do**
I'm trying to use credentials login with an existing backend. The backend API will return all information for the user including `accessToken` and `expires`. I can set the user object from the `authorize` function, but how do I also set custom `accessToken` and `expires` properties without using the generated ones.

**Documentation feedback**
*Documentation refers to searching through [online documentation](https://next-auth.js.org), code comments and issue history. The example project refers to [next-auth-example](https://github.com/iaincollins/next-auth-example).*

* [x] Found the documentation helpful
* [ ] Found documentation but was incomplete
* [x] Could not find relevant documentation
* [x] Found the example project helpful
* [ ] Did not find the example project helpful

**Comment by iaincollins** [comment]
You can [use the JWT callback](https://next-auth.js.org/configuration/callbacks#jwt) to determine what information is saved in the JSON Web Token when using the Credentials provider.

**Comment by frencojobs** [comment]
Sorry if I'm bothering a lot but I can't just find a specific tutorial or documentation with what I'm trying to do.

So I set the `expires` and `accessToken` properties in the user object and send it from `authorize` function and tried to use these properties in the jwt callback. But I'm not sure what's happening when I set the `accessToken` property of `token.account`. It doesn't seem to work.

```typescript
jwt: async (token: any) => {
  token.exp = token.user.expiresAt // this works but
  token.account.accessToken = token.user.accessToken // doesn't work, broke the session

  return Promise.resolve(token)
},
```
